index.html

```diff
@@ -73,8 +73,638 @@
+                    <p><a class="text-danger fw-bold text-decoration-none bg-grey p-2" href="#">POLITICS</a></p>
+                  </div>
+                </div>
+                <div class="img-text">
+                  <h5>
+                    <a class="fw-bold text-decoration-none text-dark" href="#">Can Joe Biden Rescind Donald Trump's Pardons?</a>
+                  </h5>
+                  <p>
+                    The president did not issue pardons for himself, his children or
+                    for former New York Mayor Rudy Giuliani.
+                  </p>
+                </div>
+              </div>
+              <div>
+                <div class="img-box">
+                  <a href="#"
+                    ><img class="feat-img" src="./images/newsweek-expert-forum.png" alt=""/></a>
+                    <div class="bottom-left">
+                      <p><a class="text-danger fw-bold text-decoration-none bg-grey p-2" href="#">EXPERTS</a></p>
+                    </div>
+                </div>
+                <div class="img-text">
+                  <h5>                
+                    <a class="fw-bold text-decoration-none text-dark" href="#"
+                      >Newsweek Announces Launch of the Newsweek Expert Forum</a
+                    >
+                  </h5>
+                  <p>
+                    Newsweek Expert Forum creates cohort of thought leaders,
+                    pioneering thinkers to share insights, expertise from range of
+                    professions, industries.
+                  </p>
+                </div>
+              </div>
+              <div>
+                <div class="img-box">
+                  <a href="#"
+                    ><img class="feat-img" src="./images/biden1.png" alt=""
+                  /></a>
+                  <div class="bottom-left">
+                    <p><a class="text-danger fw-bold text-decoration-none bg-grey p-2" href="#">POLITICS</a></p>
+                  </div>
+                </div>
+                <div class="img-text">
+                  <h5>
+                    <a class="fw-bold text-decoration-none text-dark" href="#"
+                      >Joe Biden Immigration Orders Upend Donald Trump's Legacy</a
+                    >
+                  </h5>
+                  <p>
+                    The Biden Administration will issue an order stopping
+                    construction of the border wall on its first day.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="p-2 mb-2" id="sponsored-insights-1">
+              <div>
+                <div>
+                  <div class="img-box">
+                      <a href="#"><img class="feat-img" src="./images/insights1.jpg" alt=""/></a>
+                    <div class="bottom-left nowrap">
+                      <p class="bottom-left text-primary fw-bold bg-grey p-2 ">SPONSORED INSIGHTS</p>

```

Comment:
- [ ]  Every title in this category should be an` h `tag. Please use an `h` tag as well for other titles in this section.